suite-native/transactions/src/components/TransactionsList/TransactionListGroupTitle.tsx

```diff
@@ -7,21 +7,20 @@ type TransactionListGroupProps = {
     monthKey: MonthKey;
 };
 
-const dateTextStyle = prepareNativeStyle(utils => ({
-    marginVertical: utils.spacings.sp16,
-    marginHorizontal: utils.spacings.sp16,
+const dateTextStyle = prepareNativeStyle(({ spacings }) => ({
+    marginTop: spacings.sp24,
+    marginHorizontal: spacings.sp16,
+    marginBottom: spacings.sp12,
 }));
 
 export const TransactionListGroupTitle = ({ monthKey }: TransactionListGroupProps) => {
     const { applyStyle } = useNativeStyles();
-    const { MonthNameFormatter } = useFormatters();
-    const sectionTitle =
-        monthKey === 'pending'
-            ? 'Pending'
-            : MonthNameFormatter.format(parseTransactionMonthKey(monthKey));
+    const { YearMonthFormatter } = useFormatters();
+    const date = parseTransactionMonthKey(monthKey);
+    const sectionTitle = monthKey === 'pending' ? 'Pending' : YearMonthFormatter.format(date);
```

Comment:
Can you, please, move 'Pending' to intl since we are touching this 🙏

Reply:
Yeah, noticed that after I created the PR. Fixed.